examples/chart/teleport/templates/deployment.yaml

```diff
@@ -17,14 +17,26 @@ spec:
       serviceAccount: {{ .Chart.Name }}
       containers:
       - name: teleport
-        image: quay.io/gravitational/teleport-ent:{{ .Chart.Version }}
-        imagePullPolicy: Always
-        resources:
-          requests:
-            cpu: 1m        
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        args:
+        - --insecure-no-tls
```

Comment:
TODO: Make this configurable. This can't be hard-coded. The user should decide whether to terminate TLS on Teleport Proxy or the load balancer in front of that. `--insecure-no-tls` should be optionally configured to be enabled only for the latter case.